app/_posts/2024-11-05-kuma-2-9-0.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Kuma 2.9 release with MeshTLS, Service redesign, GUI updates and more...
+description: Kuma 2.9 release with MeshTLS, Service redesign, GUI updates and more...
+date: 2024-11-06
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We're excited to announce the biggest release of Kuma since 2.0! 
+In this release we offer a complete beta release of our service rework and namespaced policies.
+
+# Notable features
+In Kuma 2.9.x we've focused on 3 main areas:
+* `MeshTLS` which provides a more granular way to roll out strict Mutual TLS.
+* New resources `MeshService` and `MeshMultiZoneService` which provide a more intuitive and powerful way to manage services in your mesh.
+* A complete rework of our destination policy attachment which works with the new services. 
```

Comment:
should we mention producer/consumer policies inn notable features, it changes how drastically how polcies are merged and applied to services